tests/cases/fourslash/codeFixInferFromUsageExistingJSDoc.ts

```diff
@@ -0,0 +1,93 @@
+/// <reference path='fourslash.ts' />
+// @allowJs: true
+// @checkJs: true
+// @noImplicitAny: true
+// @strictNullChecks: false
+// @Filename: important.js
+
+/////** @param x no types here! */
+/////**
+//// * 1
+//// * @param x a duplicate!
+//// * @param y yy
+//// */
+/////**
+//// * 2
+//// * @param z zz
+//// */
+////function f(x) {
+////    return x * 1
+////}
+////
+////var o = {
+////    /** 1
+////     * @return First one
+////     */
+////    /** 2
```

Comment:
What would happen if there were a non-JS-doc comment between these?

Reply:
That’s a decision made early by the scanner or parser, when deciding what to parse and subsequently attach to the AST. I’ll check to see what happens — I guess that the non-jsdoc comment will just be skipped but I don’t know.

Reply:
Yes, the non-jsdoc comment is deleted.

Reply:
Probably should make insertJsdocCommentBefore generate a series of deletes instead of replaceNodeRange.